### PR TITLE
Use CodeArtifact when on CodeBuild

### PR DESCRIPTION
**Pull request by abrooksv**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -66,8 +61,19 @@ repositories {
...
+                url = uri(it)
+                credentials {
+                    username = "aws"
+                    password = System.getenv("CODEARTIFACT_AUTH_TOKEN")
+                }
+            }
+        }
+        gradlePluginPortal()
         mavenCentral()
+        mavenLocal()
```
Comment (hunterwerlla) [inline build.gradle.kts:76]: mevenLocal should go first, right? Or does the order not matter

Comment (abrooksv) [reply build.gradle.kts:76]: Why does it need to go first?

Comment (abrooksv) [reply build.gradle.kts:76]: Order matters for the order it is checked in, local going first only matters if you need to use a custom / patched version to override what would be in maven central.

Comment (hunterwerlla) [reply build.gradle.kts:76]: Yes, which means we should always put it first just in case we need it? Like if we want to test a common package before?

Comment (abrooksv) [reply build.gradle.kts:76]: Fair

**Review by hunterwerlla — DISMISSED** [review]

**Review by hunterwerlla — APPROVED** [review]

**Pull request merged**